colorizer_data/converter.py

```diff
@@ -384,18 +367,60 @@ def _validate_manifest(writer: ColorizerDatasetWriter):
         )
 
 
+def _get_frame_count_from_3d_source(
+    source: Union[str, List[str], None], fallback: int = 0
+) -> int:
+    if source is None:
+        raise ValueError(
+            "Could not compute number of frames in 3D frame source: 3D frame source is `None`."
+        )
+    if isinstance(source, list):
+        source = source[0]
+    # Attempt to read the image to get info (such as length)
+    try:
+        img = BioImage(source)
+        dims = img.shape
+        # Assumes TCXYZ ordering of dimensions
+        return int(dims[0])
```

Comment:
nit: there is a more robust way by using Dimensions.T in bioio (if you want to avoid the assumption) but this is ok for now.